README.md

```diff
@@ -12,8 +12,7 @@ This package supports varying levels of functionality depending on the version
 of Go used during compilation. The `Listener` and `Conn` types produced by this
 package are backed by non-blocking I/O, in order to integrate with Go's runtime
 network poller in Go 1.11+. Additional functionality is available starting in Go
-1.12+. Go 1.10 and prior versions of Go **are not supported** and will produce
-a compilation error when trying to build this package.
+1.12+.
```

Comment:
Can you expand this documentation to indicate that 1.10 is supported in a blocking-only mode?